docs/start_doc_server.sh

```diff
@@ -11,4 +11,11 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-python -m SimpleHTTPServer 8000
+python -c "import SimpleHTTPServer" 2> /dev/null 
```

Comment:
What if we checked the version explicitly?

For instance: https://stackoverflow.com/a/12121088/4853563